R/api.R

```diff
@@ -4,6 +4,8 @@
 ##'
 ##' @param root Orderly root
 ##'
+##' @param repositories Path in which Git repositories are cloned
```

Comment:
i can see people misunderstanding this as a list of repositories instead of the path of repos, can call it `repositories_dir` anything like that works, i see it in a lot of places though so your choice if you can be bothered but i think its clearer 
```suggestion
##' @param repos_dir Path in which Git repositories are cloned
```

Reply:
Yeah good point. Will change that.